### PR TITLE
Rename cmd/test-pr to cmd/line. Add some docs.

### PR DESCRIPTION
**Pull request by spxtr**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/727)

### CODE REVIEW

**Comment by ixdy** [comment]
I really can't decide whether I hate these puns or not.

**Comment by spxtr** [comment]
I just couldn't help it.

File: ciongke/Makefile [hunk 1]
```diff
@@ -32,10 +32,13 @@ JENKINS_ADDRESS_FILE = ${HOME}/jenkins-address
 # Useful rules:
 # - create-cluster turns up a cluster then prints out the webhook address.
 # - update-cluster pushes new image versions and updates the deployment.
+# - update-jobs updates the configmap defining the Jenkins jobs.
 # - hook-image builds and pushes the hook image.
 # - hook-deployment updates the deployment.
 # - hook-service create the hook service.
-# - test-pr-image builds and pushes the test-pr image.
+# - line-image builds and pushes the line image.
```
Comment (ixdy) [inline ciongke/Makefile]: the name is clever, but I feel like it doesn't do as good a job at being obvious like "hook" and "sinker" do.

File: ciongke/cmd/line/Dockerfile [hunk 2]
```diff
@@ -15,12 +15,7 @@
 FROM alpine:3.4
 MAINTAINER spxtr@google.com
 
-RUN apk add --no-cache git python ca-certificates && update-ca-certificates
+RUN apk add --no-cache ca-certificates && update-ca-certificates
 
-RUN git config --global user.email "k8s.production.user@gmail.com"
```
Comment (ixdy) [inline ciongke/cmd/line/Dockerfile]: why are these no longer necessary?

File: ciongke/cmd/hook/main.go [hunk 3]
```diff
@@ -39,7 +39,7 @@ var (
 
 	jobConfig = flag.String("job-config", "/etc/jobs/jobs", "Path to job config file.")
 
-	testPRImage = flag.String("test-pr-image", "", "Image to use for testing PRs.")
+	testPRImage = flag.String("line-image", "", "Image to use for testing PRs.")
```
Comment (ixdy) [inline ciongke/cmd/hook/main.go]: not changing the name of the variable?

File: ciongke/cmd/line/Dockerfile (hunk 2 above)
Comment (spxtr) [inline ciongke/cmd/line/Dockerfile]: They're left over from when this did the actual git checkout.

File: ciongke/Makefile (hunk 1 above)
Comment (spxtr) [inline ciongke/Makefile]: It's a single github status line :)

File: ciongke/cmd/hook/main.go (hunk 3 above)
Comment (spxtr) [inline ciongke/cmd/hook/main.go]: Done.

File: ciongke/README.md [hunk 4]
```diff
@@ -5,8 +5,9 @@
    webhooks (`cmd/hook`). When an event of interest comes in, such as a new PR
    or an "ok to test" comment, we check that it's safe to test, and then start
    up the corresponding jobs.
-1. The jobs themselves (`cmd/test-pr`) start and watch the Jenkins job, setting
+1. The jobs themselves (`cmd/line`) start and watch the Jenkins job, setting
    the GitHub status along the way.
```
Comment (ixdy) [inline ciongke/README.md]: add "line" after "status" then? :)

Comment (spxtr) [inline ciongke/README.md]: Done :)

**Pull request merged**